services/rest/app/controllers/PerformanceController.java

```diff
@@ -127,28 +127,12 @@ public Result get(String projectId) {
       }
     }
     
-    PageList<AbstractJob<?>> pageJobs = executorService.query(new BasicDBObject("project_id", projectId));
-    pageJobs.setSortable(new MapBuilder<String, Boolean>("created_date", false).build());
-    
-    ArrayNode arrayJob = Json.newObject().arrayNode();
-    while (pageJobs.hasNext()) {
-      List<AbstractJob<?>> listJobs = pageJobs.next();
-      
-      for (AbstractJob<?> job : listJobs) {
-        
-        job.put("created_date", formater.format(job.getCreatedDate()));
-        job.put("creator", project.getCreator().getId());
-        arrayJob.add(Json.parse(job.toString()));
-      }
-    }
-    
     PageList<AbstractJob<?>> jobFirstList = executorService.query(new BasicDBObject("project_id", project.getId()), 1);

```

Comment:
what is jobFirstList variable? This name is not make sense for me. Plz, reword it.